ecc-functions/owners-approvetransfer/__init__.py

```
@@ -0,0 +1,13 @@
+import os, json, azure.functions as func, os as _os
+
+ADMIN = os.environ.get("ADMIN_SYNC_TOKEN")
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    if not ADMIN or req.headers.get("x-admin-token") != ADMIN:
+        return func.HttpResponse(json.dumps({"error": "unauthorized"}), status_code=401, mimetype="application/json")
+    try:
+        body = req.get_json() if req.get_body() else {}
+        step = _os.path.basename(_os.path.dirname(__file__)).split("owners-")[-1]
+        return func.HttpResponse(json.dumps({"ok": True, "step": step, "transferId": body.get("transferId")}), mimetype="application/json")
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)